openmetadata-service/src/main/java/org/openmetadata/service/search/elasticSearch/ElasticSearchClientImpl.java

```diff
@@ -363,6 +363,29 @@ public Response search(ElasticSearchRequest request) throws IOException {
     return Response.status(OK).entity(response).build();
   }
 
+  /**
+   * @param sourceUrl
+   * @return
+   */
+  @Override
+  public Response searchBySourceUrl(String sourceUrl) throws IOException {
+    String[] indexes =
+        new String[] {
+          ElasticSearchIndexDefinition.ElasticSearchIndexType.TABLE_SEARCH_INDEX.indexName,
```

Comment:
we should create alias for the indexes and search across all the indexes at once using the alias

Reply:
thanks for review, will push the changes..